service/history/historyEngine.go

```diff
@@ -592,7 +595,7 @@ Update_History_Loop:
 
 				// If the decision has more than one completion event than just pick the first one
 				if isComplete {
-					e.metricsClient.IncCounter(metrics.HistoryMultipleCompletionDecisionsScope,
+					e.metricsClient.IncCounter(metrics.HistoryRespondDecisionTaskCompletedScope,
```

Comment:
other counters that might be interesting in this method:
* count per decisionType
* count optimistic concurrency update failures (towards the end of the method) (or) count retries
* on failDecision=true, either emit a metric per failureCause or modify the log to print out the failure cause